.github/workflows/build.yml

```diff
@@ -389,7 +389,7 @@ jobs:
   build-image:
     needs: [build-package]
 
-    if: ${{ github.event_name != 'pull_request' }}
+    # if: ${{ github.event_name != 'pull_request' }}
```

Comment:
uncomment before merging?

Reply:
Good catch. I thought I had pushed that fix at the same time as I had marked this for review. Fixed now.